feat(js): Document new bundle varieties (#4842)
getsentry/sentry-docs: src/platforms/javascript/common/install/cdn.mdx

import JsBundleList from "~src/components/jsBundleList";

> Sentry supports loading the JavaScript SDK via a CDN. Generally we suggest using our npm package (`@sentry/browser`) instead, as using the CDN can create scenarios where Sentry is unable to load due to networking issues or common extensions like ad blockers. If you _must_ use a CDN, take a look at [loading Sentry lazily with our JS loader](../lazy-load-sentry/), which provides a deferred version of our minified ES5 browser bundle. To see what other bundles are available, see [Available Bundles](#available-bundles) below.

Reviewer: ```suggestion
Sentry supports loading the JavaScript SDK using a CDN. Generally we suggest using our npm package (`@sentry/browser`) instead, as using the CDN can create scenarios where Sentry is unable to load due to networking issues or common extensions like ad blockers. If you _must_ use a CDN, take a look at [loading Sentry lazily with our JS loader](../lazy-load-sentry/), which provides a deferred version of our minified ES5 browser bundle. To see what other bundles are available, see [Available Bundles](#available-bundles) below.
```

## Available Bundles

Our CDN hosts a variety of bundles:

- `@sentry/browser` (named `bundle.<modifiers>.js`)
- `@sentry/browser` and `@sentry/tracing` together (named `bundle.tracing.<modifiers>.js`)
- each of the integrations in `@sentry/integrations` (named `<integration-name>.<modifiers>.js`)

> Each bundle is offered in both ES5 and ES6 versions, and for each version there are three bundle varieties: unminified, minified, and minified with debug logging. (That last version can be helpful for times when you need to debug an issue which only occurs in prod. In dev, it makes most sense to use the unminified bundle, which always includes logging.)

Reviewer: That's a long parenthetical and I think you could start it later. 

```suggestion
Each bundle is offered in both ES5 and ES6 versions, and for each version there are three bundle varieties: unminified, minified, and minified with debug logging. The last version can be helpful for times when you need to debug an issue that only occurs in production. (In a development environment, it makes most sense to use the unminified bundle, which always includes logging.)
```

> So, for example, `bundle.tracing.es6.debug.min.js` is `@sentry/browser` and `@sentry/tracing` bundled together, compiled to ES6, and minified (but with debug logging included). On the simpler end of the naming spectrum, `bundle.js` is `@sentry/browser`, compiled to ES5 but not minified (with debug logging included, as it is for all unminified bundles).

Reviewer: Once this is rendered, it's a bit hard to read, so suggesting the following formatting.

```suggestion
For example, `bundle.tracing.es6.debug.min.js` is:

- `@sentry/browser` and `@sentry/tracing` bundled together
- compiled to ES6
- minified, with debug logging included

A simpler example of this naming system would be `bundle.js`, which is:

- `@sentry/browser`
- compiled to ES5
- not minified, with debug logging included (as it is for all unminified bundles)

```
